app/init/global_event_handler.ts

```diff
@@ -47,9 +48,11 @@ class GlobalEventHandler {
     };
 
     onDeepLink = (event: LinkingCallbackArg) => {
+        // TODO: this is called when returning from the SSO webview and
```

Comment:
please fix not just comment it out, do check the deeplink and make sure to handle the case